### PR TITLE
Just a small fix of the foundation link

### PR DESCRIPTION
**Pull request by ghtmtt**

### CODE REVIEW

**Comment by ghtmtt** [comment]
The commit is super small and just fixes the link of the founding project of the R files.. Can I merge it?

**Comment by yjacolin** [comment]
there is 10 commit but only url changed indeed. Did you have some problem with a rebase or so?

Merge as soon as the checks end.

**Comment by ghtmtt** [comment]
ups..you are right.. should I just rebase my repo and the problem will fix itself?
Thanks Yves

**Comment by yjacolin** [comment]
Humm, you worked on master :/ so I am really not sure. You can try a rebase but I am pretty sure it won't work as some of your commits have already been merged into master (so they appear in your PR but not in the diff with master). So you need to start with a clean local master branch:
```
git checkout -b save master # save your work in a "save" named branch
git fetch upstream # update the remote upstream
git reset --hard upstream/master # you lost everything in your local master branch, but you will start with a clean upstream./master copy in your local master branch
```
After get a clean local master branch, work as usual:
```
git checkout -b urlTypo # always create a new branch, here urlTypo
vim ... # edit again your file
git status
git add <files> # add your files in the stack
git status
git commit
git push origin urlTypo
git checkout master # I think it is a good practice to go back to master
```

Later, when your PR has been merged:
```
git branch # check you are in master branch
* master
git fetch upstream # get the changes in the upstream remote
git merge upstream/master # merge new commit to your local master branch
```
And the work as usual (see above) ;)

**Comment by ghtmtt** [comment]
Wow Yves that's a kind of assistance!! Thanks a lot!
Ok I did what you suggested.. but I don't have to create the branch `url...` you just write there for the next time right? From now on I swear I will work on different branches

**Comment by yjacolin** [comment]
Never work on master branch :)

File: source/docs/training_manual/processing/r_intro.rst [hunk 1]
```diff
@@ -2,8 +2,7 @@
 
 Use R scripts in Processing
 ===========================
-
-Module contributed by Matteo Ghetta - founded by `Scuola Superiore Sant'Anna <http://www.sssup.it/landlab>`_
+Module contributed by Matteo Ghetta - founded by `Scuola Superiore Sant'Anna - FREEWAT project <http://www.freewat.eu>`_
```
Comment (DelazJ) [inline source/docs/training_manual/processing/r_intro.rst]: I like the name of the school :)

**Pull request merged**